packages/core/src/civil.ts

```diff
@@ -14,7 +14,7 @@ const debug = Debug("civil:main");
 
 export interface CivilOptions {
   web3Provider?: Web3.Provider;
-  contentProvider?: ContentProvider;
+  ContentProvider?: ContentProviderCreator;
```

Comment:
why is ContentProvider capitalized here?

Reply:
its a Class definition not an instance